README.md

```diff
@@ -57,53 +57,6 @@ with the `oc logs` command:
 
 ```console
 $ oc logs -f bc/beego-example
-Step 0 : FROM golang:1.5
```

Comment:
this whole thing has little value, and I've tried to keep it up-to-date way too many times, so removing it.